### PR TITLE
feat(components): [descriptions] render null when empty label

### LINKED ISSUE #11572: [Component] [descriptions-item] el-descriptions-item的label插槽在没有值时应该去掉
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.45`
- Element Plus Version: `2.2.30`
- Browser / OS: `版本 110.0.5481.78（正式版本） （64 位）`
- Build Tool: `Vue CLI`

### Related Component

- `el-descriptions-item`

### Reproduction Link

[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtZGVzY3JpcHRpb25zIHRpdGxlPVwiVXNlciBJbmZvXCI+XG5cbiAgICA8ZWwtZGVzY3JpcHRpb25zLWl0ZW0gXG4gICAgICA+Tm8uMTE4OCwgV3V6aG9uZyBBdmVudWUsIFd1emhvbmcgRGlzdHJpY3QsIFN1emhvdSwgSmlhbmdzdVxuICAgICAgUHJvdmluY2U8L2VsLWRlc2NyaXB0aW9ucy1pdGVtXG4gICAgPlxuICA8L2VsLWRlc2NyaXB0aW9ucz5cbjwvdGVtcGxhdGU+XG4iLCJpbXBvcnRfbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7fVxufSIsIl9vIjp7fX0=)

### Steps to reproduce

去掉el-descriptions-item 组件的label属性后插槽的相关dom仍旧存在

### What is Expected?

插槽应该消失

### What is actually happening?

插槽仍旧存在

### Additional comments

_(empty)_

image

### PR DESCRIPTION
**Pull request by wangcch**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

close https://github.com/element-plus/element-plus/issues/11572

### CODE REVIEW

**Review by ryuhangyeong — APPROVED** [review]

File: packages/components/descriptions/src/descriptions-cell.ts [hunk 1]
```diff
@@ -94,13 +94,15 @@ export default defineComponent({
...
           },
           [
-            h(
-              'span',
-              {
-                class: [ns.e('label'), labelClassName],
-              },
-              label
-            ),
+            label != null
```
Comment (jw-foss) [inline packages/components/descriptions/src/descriptions-cell.ts:97]: We can utilize `_.isNil` in this.

```ts
import { isNil } from 'lodash-unified'
```

```suggestion
            isNil(label)
```

**Review by jw-foss — APPROVED** [review]
@wangcch code LGTM in general, but I left a comment for you to consider. Let me know if you have any questions.

File: packages/components/descriptions/src/descriptions-cell.ts (hunk 1 above)
Comment (wangcch) [reply packages/components/descriptions/src/descriptions-cell.ts]: ok. updated 👀

**Review by jw-foss — APPROVED** [review]

**Pull request merged**